lesson1/src/arrays/arrayProblems.js

```diff
@@ -39,8 +46,13 @@ class ArrayProblems {
 
    This method should perform n left rotations on the array
    */
-  static rotateLeft(array, n){
-    // todo: 🙌 do magic !
+  static rotateLeft(array, n) {
```

Comment:
remember we analize  2 types of complexity (time and space). you are now using another array same size as your original array. Can you think of a way of doing this one in place (that means on the same array without creating a new one) ?